amt_test.go

```diff
@@ -762,3 +761,41 @@ func TestEmptyCIDStability(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, c1, c3)
 }
+
+func TestCidStability(t *testing.T) {
+	const count = 500
+
+	bs := cbor.NewCborStore(newMockBlocks())
+	ctx := context.Background()
+	a := NewAMT(bs)
+
+	cids := make([]cid.Cid, count)
+
+	// Add, flushing each time.
+	for i := 0; i < count; i++ {
+		// Flush just in case.
```

Comment:
You're not really flushing "just in case", are you? You're doing so to get the root?